perfkitbenchmarker/benchmarks/aerospike_benchmark.py

```diff
@@ -188,6 +189,13 @@ def ParseOutput(output):
     if FLAGS.aerospike_storage_type == DISK:
       metadata['Disk Type'] = 'Local' if FLAGS.use_local_disk else 'Remote'
     samples.append(sample.Sample('Average Latency', latency, 'ms', metadata))
+    if latency < 1.0:
+      max_throughput_for_completion_latency_under_1ms = tps
```

Comment:
Is `tps` guaranteed to be monotonically increasing (that is: could `max_throughput_for_completion_latency_under_1m` ever decrease here)?